src/installation/index.md

```diff
@@ -1,117 +1,127 @@
 # Setting Up a Development Environment
 
-With an understanding of the ecosystem surrounding Rust on Espressif chips, we can move on to actual development. If you are not aware of the two possible development approaches or do not understand the differences between writing `std` and `no_std` applications, please first read the [Ecosystem Overview] chapter.
+At the moment, Espressif SoCs are based on two different architectures -- RISC-V and Xtensa. Each architecture supports the `std` and `no_std` application development.
 
-Let's take a moment to discuss the Rust support for the different architectures of the Espressif chips and the different approaches in more detail. At this moment, Espressif SoCs are based on two different architectures: `RISC-V` and `Xtensa`. The support for those two architectures in the Rust programming language is very different.
+To set up the development environment, do the following
 
-[Ecosystem Overview]: ../overview/index.md
+1. [Install Rust][install-rust]
+2. Install requirements for the targets
+    - [RISC-V targets][risc-v-targets]
+    - [Xtensa targets][xtensa-targets]
```

Comment:
Maybe we should add an optional third step for installing std requirements

Reply:
Added a sentence after this list:

As mentioned in the installation procedures below, for `std` development also don't forget to install `std` Development Requirements.